lib/cli/cwdValidator.js

```diff
@@ -0,0 +1,33 @@
+const { consola } = require("consola");
+const path = require("path");
+const fs = require("fs");
+const fsUtils = require("../utils/fsUtils");
+
+class CwdValidator {
+  static validate() {
+    const warningMessage = `Please, double check that you are executing "silverfin" CLI in the correct directory.`;
+    const currentDirectory = process.cwd();
+    const gitDirPresent = fs.existsSync(path.join(currentDirectory, `.git`));
+
+    if (gitDirPresent) {
+      return;
+    }
```

Comment:
Should we check for git repository presence, or are there cases where we use it outside any repo? :thinking:

Reply:
I don't want to be more strict at this point as we may be blocking uses cases that we are not foreseeing now.
A warning should be more than enough for the user to correct their actions in my opinion. And we can always modify it to be more strict if we see the reason to do so in the future.